Sources/StringsLintFramework/Models/StringDict.swift

```diff
@@ -38,15 +40,30 @@ extension StringDict.PluralItem {
     struct Format: Decodable {
         enum CodingKeys: String, CodingKey {
             case formatValueKey = "NSStringFormatValueTypeKey"
+            case zero
+            case one
+            case two
+            case many
+            case few
+            case other
```

Comment:
@SarahMFaire[ brought up a good point here.](https://github.com/Faire/android/pull/8154#issuecomment-1282625181) I wonder if we should enforce that we always provide one, many and others? thoughts @SarahMFaire @tbroadley @rlnucci ?

Reply:
what is the difference btw `many` and `other` BTW 😂

Reply:
I think it would make sense to enforce that!

Side note: can I ask why both "many" and "other" are necessary? If we enforce that we always provide "one", "many", and "other", I guess we'd only use "other" for the case where the number we're passing into the string is "zero". So maybe we should enforce that we always provide "zero", "one", and "many"?

Reply:
Looking at the resource Sarah linked, 2 and 1M are translated differently

<img width="1643" alt="Screen Shot 2022-10-19 at 1 08 32 PM" src="https://user-images.githubusercontent.com/37663461/196758463-25273494-7b24-4e54-bc46-e7b8dbdc3d74.png">

Reply:
Ah let me find the link to the standard. "Other" does not mean many. Certain languages have a different treatment of nouns if it's a large number versus a small number. Like "1 000 000 shoes" would spell shoes different then "3 shoes". Other is just the catchall for everything else if the language doesn't have a specific case. So English only has "one" as a special case. Then every other value (0, 2, 3, 4, ...) falls into "Other". Chinese for example doesn't have any special cases, there are no plurals forms and they would only need to provide the "other" category which would be used for everything. You can find the explanation of all these cases here: https://cldr.unicode.org/index/cldr-spec/plural-rules

Reply:
So enforcement would depend on language. For example, according to this standard, French should have a "many" case and Android gives us a warning that our plurals don't contain that. We're surpressing that because the translators don't seem concerned.

Reply:
sounds like we ideally shouldn't just mirror the English plural rules to any other language and submit different jsons to be translated. e.g. even tho in English we have this rule
```
{one: "SKU", other: "SKUs"}
```

In french we should submit a different JSON to the translation service
```
{one: "xxx", many: "xxx", other: "xxx"}
```
Maybe this is something we should check with our language experts?